src/utils/hibp.js

```diff
@@ -146,6 +146,37 @@ async function loadBreachesIntoApp (app) {
   }
   log.info('done-loading-breaches', 'great success 👍')
 }
+
+/**
+Get addresses and language from either subscribers or email_addresses fields:
+TODO: Add route
```

Comment:
It might just be that I don't have enough clarity on the codebase and project timelines yet, but possibly this comment could use some clarification on what actually needs to happen?

Reply:
Oh, that one is a WIP comment that I’ve missed — thx for catching it. It’s not relevant anymore.